src/aura/StatusBoardTemplate/StatusBoardTemplate.cmp

```diff
@@ -0,0 +1 @@
+<aura:component ></aura:component>
```

Comment:
@agustin-lessa I guess this component is not being used?

Reply:
I think this is here because at first i thought about doing a custom page template but then i decided against it, so this is probably me creating the template but doing nothing on it and later forgetting it was there.